### PR TITLE
Add version flag support on model export command

### LINKED ISSUE #11925: [mesheryctl] Add version flag support on model export command
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

### Current Behavior

mesheryctl model export only latest version

### Desired Behavior

We want to be able to export a specific version of a model

### Implementation

We can add a flag `version`

### Acceptance Tests

If we pass the flag `version` the specified version of the model is exported

```bash
mesheryctl model export --version x.x.x
```
 
### Mockups

---

### Contributor Resources

- ⌨️ [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by lekaf974**
**Notes for Reviewers**

This PR fixes #11925

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by hexxdump — DISMISSED** [review]
LGTM

**Comment by Ashparshp** [comment]
Thank you @lekaf974, let's discuss this PR during the Meshery dev meeting call, if you would. Adding it as an agenda item to the meeting minutes.

Timing: 7:30 PM IST / 9:00 AM CT (Wednesday)
Meeting minutes: [Google Doc](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit)
Meeting link: [meet.layer5.io/meshery](https://meet.layer5.io/meshery)

**Review by alphaX86 — DISMISSED** [review]
Approved. LGTM!

**Review by leecalcote — COMMENTED** [review]
yay!

I wonder if it's helpful to include an example of this flag in the usage samples.

**Comment by lekaf974** [comment]
> yay!
> 
> I wonder if it's helpful to include an example of this flag in the usage samples.

Good idea, will include an example

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 1]
```diff
@@ -911,16 +911,17 @@ model:
...
-         - import model /path/to/[file.yaml|file.json]
-         - import model /path/to/models
+         - mesheryctl import model /path/to/[file.yaml|file.json]
+         - mesheryctl import model /path/to/models
     export:
       name: export
       description: Export a model by name to the given output type (default is oci).
-      usage: mesheryctl model export [ model-name ]
+      usage: mesheryctl model export [model-name]
       examples:
```
Comment (lekaf974) [inline docs/_data/mesheryctlcommands/cmds.yml:920]: @leecalcote: Are these examples field really needed, it seems not to be used ?

**Comment by lekaf974** [comment]
Added example

**Comment by l5io** [comment]
🚀 Preview for commit 7d9ef8833de0dbfa30b85d227c848e9b9df1d0d5 at: https://66e1859c3795d7fdad76700d--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
@lekaf974 
Thank you for your contributions.
Let's discuss this PR on Meshery development call today. Adding it as an agenda item to the meeting minutes.

Meeting minutes: [Google Docs](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit)
Meeting link: [meet.layer5.io/meshery](https://meet.layer5.io/meshery)

File: docs/_data/mesheryctlcommands/cmds.yml (hunk 1 above)
Comment (alphaX86) [reply docs/_data/mesheryctlcommands/cmds.yml:920]: Hi @lekaf974, let me answer this one.

In early versions of docs, we used this as no auto-doc functionality was brought in at that time. Eventually when we brought the auto-doc function (via Cobra), the command index page displays only name and description. To answer your Q, yes, it is not being used for now as it is taken care within command doc pages.

Comment (alphaX86) [reply docs/_data/mesheryctlcommands/cmds.yml:920]: If you included examples in the respective golang file, then the example field in the command doc page will auto populate.

**Review by alphaX86 — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 433d73d2b40ebbb450f33c1a17f12102ccdaef99 at: https://66e253f629dd6cc807f5dbab--meshery-docs-preview.netlify.app

**Pull request merged**